Mobile/starter_project_dartsmiths/lib/onboarding/presentation/widgets/slide_dots.dart

```diff
@@ -0,0 +1,33 @@
+import "package:flutter/material.dart";
+import '../../../core/utils/colors.dart';
+
+// This widget acceps index and returns pagination dots making the index active
+class SlideDotes extends StatefulWidget {
```

Comment:
it is always better to think of what the widget is for so that the naming will be easier
rename this as well to StepperDots or something else of some sort